etcdmain/config.go

```diff
@@ -293,6 +297,9 @@ func (cfg *config) Parse(arguments []string) error {
 	if 5*cfg.TickMs > cfg.ElectionMs {
 		return fmt.Errorf("-election-timeout[%vms] should be at least as 5 times as -heartbeat-interval[%vms]", cfg.ElectionMs, cfg.TickMs)
 	}
+	if cfg.ElectionMs > maxElectionMs {
+		return fmt.Errorf("-election-timeout[%vms] is unreasonably long, and should be set less than %vms", cfg.ElectionMs, maxElectionMs)
```

Comment:
i do not like the wording here again. `unreasonably` sounds too harsh. Can we simply change it to `too long`? Also this is too verbose.